docs/gettinginvolved/infrastructure/documentation.rst

```diff
@@ -31,9 +31,11 @@ obsolete releases that are no longer hosted are permanently redirected to the la
      <!-- Marketing Release Documentation -->
      <RoutingRule>
```

Comment:
Above this is the line "The following configuration is a sample configuration, with versions current as of 2015-01-28". This date should be changed.

However, since changing the routing rules is an automated process, I'm not convinced of the value of this part of the documentation.